src/components/Dashboard/TodoList.vue

```diff
@@ -1172,6 +1175,54 @@ export default class TodoList extends Mixins(
     }
   }
 
+  private async loadChangeOfRegistration (task: ApiTaskIF): Promise<void> {
+    const filing = task.task.filing
+    const header = filing.header
+    const changeOfRegistration = filing.changeOfRegistration
+
+    // NB: don't check "changeOfRegistration" as it may be empty
```

Comment:
Is this true or did you clone it from another function?

Reply:
mmmmm no, it shouldn't be true actually. 
It's a copy - paste - clone error.
Removed!

Reply:
Good idea 👍